security/botan/Portfile

```diff
@@ -23,20 +24,20 @@ long_description    Botan is a C++ library implementing a variety of \
 homepage            https://botan.randombit.net
 master_sites        ${homepage}/releases/
 distname            Botan-${version}
-extract.suffix      .tgz
+extract.suffix      .tar.xz
```

Comment:
then you probably should set `use_xz yes` instead

Reply:
Thanks, changed.